bot/exts/evergreen/issues.py

```diff
@@ -159,79 +196,74 @@ async def issue(
             user: str = "python-discord"
     ) -> None:
         """Command to retrieve issue(s) from a GitHub repository."""
-        if not ctx.guild or not(
-            ctx.channel.category.id in WHITELISTED_CATEGORIES
-            or ctx.channel.id in WHITELISTED_CHANNELS
-        ):
-            await ctx.send(
-                embed=discord.Embed(
-                    title=random.choice(NEGATIVE_REPLIES),
-                    description=(
-                        "You can't run this command in this channel. "
-                        f"Try again in <#{Channels.community_bot_commands}>"
-                    ),
-                    colour=discord.Colour.red()
-                )
-            )
-            return
-
-        result = await self.fetch_issues(set(numbers), repository, user)
-
-        if result == FetchIssueErrors.value_error:
-            await ctx.invoke(self.bot.get_command('help'), 'issue')
+        # Remove duplicates
+        numbers = set(numbers)
 
-        elif result == FetchIssueErrors.max_requests:
+        if len(numbers) > MAXIMUM_ISSUES:
             embed = discord.Embed(
                 title=random.choice(ERROR_REPLIES),
                 color=Colours.soft_red,
-                description=f"Too many issues/PRs! (maximum of {MAX_REQUESTS})"
+                description=f"Too many issues/PRs! (maximum of {MAXIMUM_ISSUES})"
             )
             await ctx.send(embed=embed)
+            await ctx.invoke(self.bot.get_command('help'), 'issue')
```

Comment:
This can use [`bot.utils.extensions.invoke_help_command`](https://github.com/python-discord/sir-lancebot/blob/main/bot/utils/extensions.py#L36-L42) instead.